data/remnant/remnant 2 side missions.txt

```diff
@@ -640,3 +640,59 @@ mission "Remnant: Shattered Light 6"
 			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
 			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
 			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`
+
+
+
+mission "Remnant: Scent of Rain 1"
+	name "Search for the unknown Korath ship."
+	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."
+	landing
+	source
+		government "Remnant"
+	destination "Caelian"
+	to offer
+		has "Remnant: Cognizance 25: done"
+	npc "scan outfits"
+		government "Korath"
+		personality waiting target
+		system
+			system "Kor Zena'i"
+		ship "Tubfalet" "Ra'puk At 295"
+		dialog `In poor lighting this ship might have passed for a Palavret, but major structural difference suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`
+	npc
+		government "Korath"
+		personality waiting
+		system
+			system "Kor Zena'i"
+		fleet
+			names "korath"
+			cargo 3
+			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
+			outfitters "Korath Basics" "Korath Exiles"
+			variant
+				"'olofez" 3
+	on offer
+		event "remnant: tubfalet appearance" 20 31
```

Comment:
It seems like it'd be possible to just not complete this mission until this event happens, at which point, it seems a bit silly to still require us to scan one particular ship, especially when they will be venturing all the way to the Remnant home systems on raids. This mission should probably just have a deadline or otherwise end before we get to this event.

Reply:
It might be a good idea to mention that in the `on complete` conversation, then, instead of having him act like this is the first scan he's seen of one of these ships.

Reply:
Well, with the description as written, it fits several situations:
1. This is the first good scan the Remnant have received of it.
2. Chilia is being polite and/or respectful of the pilot's efforts (like any good military commander: If a long range scouts goes above and beyond scouting the enemy's forces and brings back detailed imagery of a new weapon, they listen and read the report and appreciate it - not just going "oh, I already got a report on that.")
3. Chilia doesn't want to reveal that they already have scans of it.

The main situation it *doesn't* accommodate is:
4. The Remnant already have scans, and want to either show off to the player or "put them in their place" by revealing that, no matter how fast the player got them, the Remnant *already* have scans.

In this particular situation, 1 & 2 are the primary alternatives I want to account for. I do *not* want to risk implying 4 here.

Reply:
> It might be a good idea to mention that in the `on complete` conversation, then, instead of having him act like this is the first scan he's seen of one of these ships.

@warp-core I didn't actually state as much in my previous post, but I did adjust the on complete to show that Chilia is looking at a scan before the player arrives. He doesn't elaborate on whether that's the pilot's data transmitted ahead, or from another source. Leaving it imprecise can let the pilot imagine whatever fits best to them. Does this work for you?